playground/index.html

```diff
@@ -1,58 +1,146 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   <title><%= htmlWebpackPlugin.options.title %></title>
+  <link rel="icon" type="image/x-icon" href="<%= require('../resources/favicon.ico').default %>" />
   <style>
-    html, body {
+    /* The full-height solution for IE 11 is taken from https://gist.github.com/palicko/26b327cd2198e52c25b13485b4dd02bc */
+    html {
       display: flex;
-      min-height: 100%;
-      flex: 1;
     }
     body {
-      color: #333;
-      font-family: sans-serif;
+      display: flex;
+      flex-direction: column;
+      width: 100%;
+      min-height: 100vh;
       padding: 0;
+      margin: 0;
+      font-family: sans-serif;
+    }
+    .content {
+      flex: 1;
+      width: 100%;
+      max-width: 60em;
+      box-sizing: border-box;
+      margin: 0 auto;
+      padding: 1.25em;
+      display: flex;
+      flex-direction: column;
+    }
+    header {
+      flex: none;
+      margin-bottom: 1.2vh;
     }
-    main {
-      margin: auto;
-      max-width: 800px;
-      padding: 20px;
+    header img {
+      height: 5vh;
+      max-width: 100%;
+    }
+    button {
+      border: none;
+      background: #f04405;
+      color: #fff;
+      font: inherit;
+      font-size: 88%;
+      font-weight: 600;
+      line-height: 1.2;
+      border-radius: 0.57em;
+      cursor: pointer;
+      margin: 0;
+      padding: 0.85em 1.7em;
       text-align: center;
+      outline: none;
+      transition: all 0.25s ease-out;
+    }
+    button:hover {
+      background: #8c2703;
     }
-    h1 {
-      margin: 0 0 1.3em;
-      color: #393e9e;
+    button:active {
+      background: #cd3f10;
     }
-    .visitorIdHeader {
-      font-size: 16px;
-      color: #999;
+    button:disabled {
+      cursor: not-allowed;
+      background: #e9987c;
     }
-    .visitorId {
-      font-size: 24px;
-      font-weight: 500;
+    .buttons {
+      flex: none;
+    }
+    .buttons button {
+      margin: 0 0.8em 0.8em 0;
+    }
+    .outputHolder {
+      flex: 1;
+      min-height: 4em;
+      position: relative;
+    }
+    .output {
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100%;
+      height: 100%;
+      box-sizing: border-box;
+      background: #282c34;
+      box-shadow: 0 0 1.25em rgba(0, 0, 0, 0.2);
+      border-radius: 0.7em;
+      padding: 1em 1.25em;
+      color: #e1e5ea;
+      overflow: auto;
+    }
+    .output pre {
+      padding: 0;
+      margin: 0;
+      font-size: 123%;
```

Comment:
it seems a bit strange that we would use number other than 100% for default and then do 100% for small. why not do 100% for default, 75% for small and 200% for big? I'm probably missing something, so I'd appreciate if you explained your logic.

Reply:
The parent font size is `16px`. If a child `pre` font size is set to `100%`, the result font size is `13px` for some reason. This phenomenon persists in different browsers. I want `16px` size for the default so I've set `font-size: 123%`.

Reply:
Agree seems strange. Maybe it would be easier to use rem values? 

I also wouldn't use em's for layout values (padding, margins) due to potential cascading issues, but it looks like it would be too much work to change it now and it looks good now, so it's fine.

Reply:
@Lisenish `rem` is affected by this phenomenon too. If I set `font-size: 1rem;`, the `pre` (and only `pre`) font size will be `13px`.